examples/basic_bq/README.md

```diff
@@ -0,0 +1,7 @@
+# terraform-google-bigquery example
+
+## Run example
+`cd ./examples/basic_bq`
+`terraform init`
+`terraform plan -var-file example.tfvars`
```

Comment:
It looks like if you use the `-var-file` syntax you need to provide an "=" sign, but that might be optional? https://www.terraform.io/docs/commands/plan.html#var-file-foo